java/src/main/java/org/rocksdb/DBOptions.java

```diff
@@ -759,6 +759,19 @@ public long compactionReadaheadSize() {
     return compactionReadaheadSize(nativeHandle_);
   }
 
+  @Override
+  public DBOptions setDailyOffpeakTimeUTC(String offpeakTimeUTC) {
```

Comment:
I think this is good design. Somebody can say that _java date/time_ data structures can be better, but conversion will probably make more problems. 👍

Reply:
I thought hard about that. If I would use the the java date/time, I need to parse the HH:mm-HH:mm format.